dashboard/modules/job/job_head.py

```diff
@@ -143,7 +143,13 @@ async def close(self, ignore_error=True):
 
 
 class JobHead(dashboard_utils.DashboardHeadModule):
-    """Runs on the head node of a Ray cluster and handles Ray Jobs APIs."""
+    """Runs on the head node of a Ray cluster and handles Ray Jobs APIs.
+
+    NOTE(architkulkarni): Please keep this class in sync with the OpenAPI spec at
+    `doc/source/cluster/running-applications/job-submission/openapi.yml`.
+    We currently do not automatically check that the OpenAPI
+    spec is in sync with the implementation.
```

Comment:
I wouldn't know how to do this from the description alone, can you include instructions?

Reply:
@edoakes updated, let me know if any parts are unclear